src/controller/python/chip/clusters/Attribute.py

```diff
@@ -711,10 +720,10 @@ def handleEventData(self, header: EventHeader, path: EventPath, data: bytes, sta
     def handleError(self, chipError: PyChipError):
         self._resultError = chipError.code
 
-    def _handleSubscriptionEstablished(self, subscriptionId):
+    def _handleSubscriptionEstablished(self, subscriptionId, maxInterval):
         if not self._future.done():
             self._subscription_handler = SubscriptionTransaction(
-                self, subscriptionId, self._devCtrl)
+                self, subscriptionId, maxInterval, self._devCtrl)
```

Comment:
So... In this specific spot, `self._pReadClient` is the ReadClient and we can just get the maxInterval from it as needed, either here or in the `SubscriptionTransaction`.  So there's no need to add new API here at all...